language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/hover/HoverUtil.java

```diff
@@ -349,6 +351,31 @@ private static Hover getFunctionHoverMarkupContent(FunctionSymbol symbol, HoverC
         return hover;
     }
 
+    private static Hover getVariableHoverMarkupContent(VariableSymbol symbol) {
+        Optional<Documentation> documentation = symbol.documentation();
+        if (documentation.isEmpty()) {
+            return getDefaultHoverObject();
+        }
+
+        List<String> hoverContent = new ArrayList<>();
+        if (documentation.get().description().isPresent()) {
+            hoverContent.add(documentation.get().description().get());
+        }
+        TypeSymbol varTypeSymbol = CommonUtil.getRawType(symbol.typeDescriptor());
```

Comment:
Let's not get the raw type here. Let's stick to the visible type descriptor since that is usable more than the raw type